test/module/shared_model/validators/protobuf/proto_block_validator_test.cpp

```diff
@@ -105,6 +103,5 @@ TEST_F(ProtoBlockValidatorTest, BlockWithInvalidPrevHash) {
   versioned_block.mutable_payload()->set_prev_block_hash("not_hex");
   *invalid_block.mutable_block_v1() = versioned_block;
 
-  auto answer = validator.validate(invalid_block);
-  ASSERT_TRUE(answer.hasErrors()) << invalid_block.DebugString();
+  ASSERT_TRUE(validator.validate(invalid_block));
```

Comment:
Maybe `<< invalid_block.DebugString();` was useful here?

Reply:
the old checks would just print "got: false, expected: true". but the new errors are streamable, so they would print the complete error info.